dashboard/apps/prototype/views.py

```diff
@@ -4,6 +4,76 @@
 from django.shortcuts import render
 from django.http import JsonResponse
 
+from .models import Person, Project, Client, Task, Role, Rate
+
+
+def get_total_times(projects):
+
+    project_days = []
+    project_names = []
+
+    for project in projects:
+        tasks = project.tasks.all()
```

Comment:
logic for getting the man days on a project will potentially be very reusable therefore it is better placed on `models.Project`. it also lends itself for easier testing. 

here is an example for task:

https://github.com/ministryofjustice/moj-product-dashboard/blob/feature/calculation/dashboard/apps/prototype/models.py#L87
https://github.com/ministryofjustice/moj-product-dashboard/blob/feature/calculation/dashboard/apps/prototype/models.py#L94